src/Playroom/Playroom.js

```diff
@@ -352,13 +352,7 @@ export default class Playroom extends Component {
           className={styles.previewContainer}
           style={{ bottom: this.state.height }}
         >
-          <Preview
-            code={renderCode}
-            components={components}
-            themes={themes}
-            frames={frames}
-            frameComponent={frameComponent}
-          />
+          <Preview code={renderCode} themes={themes} frames={frames} />
```

Comment:
Why are `frameComponent` and `components` not passed anymore?

Reply:
They are [unused](https://github.com/seek-oss/playroom/blob/master/src/Playroom/Preview/Preview.js#L7).